### PR TITLE
feat(timm_mobilenetv2): add timm MobileNetV2 image-classification family

### PR DESCRIPTION
**Pull request by zhenshanx-nv**
## Background

MobileNetV2 is one of the remaining mobile classifier baselines in the tensorrtx
set. `timm/mobilenetv2_100.ra_in1k` cannot be built or served today.

## Exit Criteria

- A `timm_mobilenetv2` family builds timm MobileNetV2 checkpoints from HF-hosted
  safetensors and produces logits matching timm's own implementation.
- The family is registered across the runtime strategy matrix, validation
  workloads, benchmark suite, website data, and the E2E model registry.

Non-goals: quantized builds and tensor-parallel builds.

## Implementation

Block shape is recovered from the checkpoint: the block kind follows from which
convolutions are present and the depthwise kernel from its weight shape. The
activation is uniform ReLU6, so only the per-stage stride comes from an
architecture table.

MobileNetV2 carries no squeeze-excite gate. The gate is still detected from the
keys and a checkpoint that has one is **rejected** rather than built with the
gate silently dropped, which would give correct shapes and wrong numbers.

The only new op is ReLU6, expressed as a clamp to `[0, 6]`.

Structurally this family is close to `timm_mnasnet`: the same key names, the
same head order, and the same 7-stage stride schedule, differing only in the
activation. Per the repository's model-family ownership rule the code is
duplicated rather than shared.

No public API, ABI, or bundle format change. No new dependencies.

### Change categories

- [x] Model or runtime behavior
- [ ] Public API
- [ ] ABI
- [ ] Bundle or artifact format
- [ ] Dependencies
- [ ] Documentation only
- [ ] CI or developer tooling

### Commands and Results

```
python -m pytest tests/builder/ tests/tools/ tests/e2e_harness/ -q -n 8 \
  --dist=worksteal --import-mode=importlib -p no:cacheprovider
=> 4042 passed, 8 skipped

python -m pytest -q \
  tests/e2e/models/timm_mobilenetv2/test_timm_mobilenetv2_family_plugin.py
=> 15 passed

cmake --build $B --target trtmc_model_timm_mobilenetv2 \
  test_timm_mobilenetv2_image_preprocess_seam
$B/test_timm_mobilenetv2_image_preprocess_seam
=> build and link clean; test exit 0

python -m ruff check ... => All checks passed
python tools/legal_headers.py => findings=0
clang-format => clean
```

Numerical parity against timm's own implementation:

| Checkpoint | Correlation | argmax | top-5 |
| --- | --- | --- | --- |
| `timm/mobilenetv2_100.ra_in1k` | 0.99997847 | match | 5/5 |

The state dict loads into timm with no missing or unexpected keys.

The first parity attempt reported a failure that turned out to be a defect in
the **test harness**, not the model: the verification script had been derived
from the Inception one and still fed a 299x299 input, which timm accepts through
its adaptive pooling while the engine builds the 224x224 input its config
declares. Reimplementing the block order in plain PyTorch matched timm exactly
(correlation 1.0, maximum difference 0.0), which localised the fault to the
harness. Every other family's script was audited afterwards and feeds the size
its own checkpoint declares.

### Hardware, Environment, and Revisions

- GPU: NVIDIA A100-SXM4-80GB, compute capability 8.0.
- Container: `Dockerfile.dev.x86` dev image, Ubuntu 24.04, Python 3.12.
- TensorRT 11.1.0.106, CUDA architecture `80-real`, Release build.
- Reference: timm 1.0.29 with torchvision 0.27.0+cpu on torch 2.12.0+cpu.
- Parity measured at fp32; the family also supports fp16.
- `timm/mobilenetv2_100.ra_in1k` @ `5afa12513b048c79b9147a5d210e9bdc50035481`.

  The manifest does not pin `hf_revision`: the timm reference resolves
  `hf-hub:<id>` at `main`, so a pin disagrees with the cache the warm step
  populates and fails the offline reference run. See #1121.

### Not Run / Remaining Gaps

- No E2E harness run. The manifest is registered but was not executed here.
- Only the `_100` width was verified numerically. Other widths share the
  schedule and are expected to work, but none was downloaded.
- No performance numbers. The benchmark row is registered but was not run.

## Notes For Future Readers

When adding a family by adapting another one's verification script, check the
input size first. timm's adaptive pooling accepts a wrong size silently, so the
comparison fails in a way that looks like a model defect.

### Risk level

- [x] Low
- [ ] Medium
- [ ] High

Additive family. Existing families are untouched except for shared registration
points, all widened rather than redirected, and the full CPU suite passes.

### CODE REVIEW

**Comment by zhenshanx-nv** [comment]
Closing as redundant: the timm_mobilenetv2 family landed in #1178, so this adds nothing. It also targets the pre-cutover layout removed by #1188.
